en/concept/core_components.md

```diff
@@ -0,0 +1,91 @@
+| Matrix (Library)                                                                   |                                      |                                                  |
+| Autotuning (Library)                                                               |                                      |                                                  |
+| [Github Actions (Continuous Integration)][Github Actions (Continuous Integration)] |                                      |                                                  |
+| [Jenkins (Continuous Integration)][Jenkins (Continuous Integration)]               |                                      |                                                  |
+| Linux (Toolchain)                                                                  |                                      |                                                  |
+| Mac OS (Toolchain)                                                                 |                                      |                                                  |
+| [Windows (Toolchain)][Windows (Toolchain)]                                         | WSL2                                 |                                                  |
+| [ROS2][ROS2]                                                                       |                                      |                                                  |
+| [Gazebo Classic (Simulation)][Gazebo Classic (Simulation)]                         |                                      |                                                  |
+| [Gazebo (Simulation)][Gazebo (Simulation)]                                         |                                      |                                                  |
+| Flightgear (Simulation)                                                            |                                      |                                                  |
+| JSBSim (Simulation)                                                                |                                      |                                                  |
+| jMAVSim (Simulation)                                                               |                                      |                                                  |
+| SIH (Simulation)                                                                   |                                      |                                                  |
+| Multivehicle Simulation                                                            |                                      |                                                  |
+| [MAVLink (Communication)][MAVLink (Communication)]                                 |                                      |                                                  |
+
+[uORB (Architecture)]: https://github.com/PX4/PX4-Autopilot/issues?q=label%3AUorb+
```

Comment:
Note, I pulled all the links out to make this table editable. This works better with automatic prettiering.